pipeline/remove_subscribers.py

```diff
@@ -1,50 +1,32 @@
 """Removing products data that no longer have active subscriptions in the RDS database."""
 import logging
 from psycopg2.extensions import connection
+from psycopg2.extras import DictCursor
 from dotenv import load_dotenv
 from connect_to_database import configure_logging, get_connection
 
 
-def get_active_subscriptions(conn: connection) -> set:
-    """Returns all product_ids that have users subscribed to it from subscription schema."""
+def get_product_ids_from_table(conn: connection, table_name: str, distinct: bool = False) -> set:
+    """Returns a set of product IDs based on specified schema."""
```

Comment:
Very nice way to slim down the code, especially leaving a default for the distinct unless stated.